subsys/fs/shell.c

```diff
@@ -895,46 +882,40 @@ static int cmd_mount_littlefs(const struct shell *sh, size_t argc, char **argv)
 
 SHELL_STATIC_SUBCMD_SET_CREATE(sub_fs_mount,
 #if defined(CONFIG_FAT_FILESYSTEM_ELM)
-	SHELL_CMD_ARG(fat, NULL,
-		      "Mount fatfs. fs mount fat <mount-point>",
-		      cmd_mount_fat, 2, 0),
+			       SHELL_CMD_ARG(fat, NULL, "Mount fatfs. fs mount fat <mount-point>",
+					     cmd_mount_fat, 2, 0),
 #endif
 
 #if defined(CONFIG_FILE_SYSTEM_LITTLEFS)
-	SHELL_CMD_ARG(littlefs, NULL,
-		      "Mount littlefs. fs mount littlefs <mount-point>",
-		      cmd_mount_littlefs, 2, 0),
+			       SHELL_CMD_ARG(littlefs, NULL,
+					     "Mount littlefs. fs mount littlefs <mount-point>",
+					     cmd_mount_littlefs, 2, 0),
```

Comment:
The original definitions were more readable, even though they were not correctly formatted either, as there is not indent expected here, in the first line of definition, as this is in the main scope of unit.

Reply:
Not sure how to respond; I simply ran clang-format of there's an issue with the auto formatter, it should be discussed and fixed, but that's probably out of scope for this change.

Reply:
Is there a guideline somewhere on how mandatory clang-format is? Is it optional?

Reply:
https://docs.zephyrproject.org/latest/contribute/style/index.html#clang-format

This is not a new code.